src/sql/parts/dashboard/services/breadcrumb.service.ts

```diff
@@ -60,6 +60,9 @@ export class BreadcrumbService implements IBreadcrumbService {
 	}
 
 	private getServerBreadcrumb(profile: ConnectionProfile): MenuItem {
+		if (profile.connectionName !== null && profile.connectionName !== undefined){
```

Comment:
inline this with a "truthy" test and use the ? operator - can maintain a single return statement.